using-timescaledb/continuous-aggregates.md

```diff
@@ -522,35 +479,39 @@ or have other areas you think we should work on, please get in touch via our
 [GitHub][timescale-github] or [Slack][support-slack].
 
 
-**Parallelized Materializations:**
-Currently, materializations are populated by a single background worker per job.
-We plan to enable multiple workers to be launched to perform a single
-materialization. This can be especially helpful in the initial population of the
-continuous aggregate, if it's created after the raw hypertable has a significant
-amount of data.
+**Parallelized Materializations:** Currently, materializations are
```

Comment:
I am wondering if we should remove the whole Future Work. Maybe replace with "Limitations".

It seems slightly odd to me to give this kind of roadmap that we aren't sure about when and if we will deliver on.

Reply:
All for removing the future work section, I do not think we should have any of those at all since they are hard to keep accurate.

Reply:
Then let's remove. At least the "Synchronous Invalidation" section doesn't seem up-to-date with recent changes so it is confusing to keep that part.